source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -394,16 +407,33 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol, SelectActionHa
     }
     
     func configureInputElements(element: ACSBaseInputElement, view: NSView) {
-        setupLabel(for: element)
+        setupLabel(for: element, view: view)
         addArrangedSubview(view)
         if view is ACRContentStackView {
             view.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
         }
         setupErrorMessage(element: element, view: view)
     }
     
-    private func setupLabel(for element: ACSBaseInputElement) {
-        guard renderConfig.supportsSchemeV1_3, let label = element.getLabel(), !label.isEmpty, inputLabelField?.stringValue != label else { return }
+    func getErrorTextField(for inputView: InputHandlingViewProtocol) -> NSTextField? {
+        guard let errorMessageField = self.errorMessageFieldMap.object(forKey: inputView.key as NSString) else {
+            logError("For show error message, field not found in MapTable :: key \(inputView.key).")
```

Comment:
Are we sure inputView.key is not PII

Reply:
yes, without that it'll not render specifically for input element.

Reply:
Yes, but we should not log data which can be sensitive
It is against PII regulation
And key is something which is part of the json, so it will be considered sensitive data